samples/sdl/main.c

```diff
@@ -18,10 +18,10 @@
 #include "string.h"
 #include <SDL.h>
 
-int _exit(int x)
+int Reboot(int x)
```

Comment:
The name of this function is misleading. It's more of a `PrintErrorAndThenQuitAfter5Seconds`.

*I can't think of a good name myself.*